data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/CountAggregateAction.java

```diff
@@ -49,6 +54,7 @@ public class CountAggregateAction implements AggregateAction {
     public final String outputFormat;
     private long startTimeNanos;
     private final String metricName;
+    private final IdentificationKeysHasher identificationKeysHasher;
```

Comment:
Maybe rename this to `uniqueKeysHasher`. It is used only for unique keys. But, the concept of identification_keys is broadly used with aggregations.